### PR TITLE
Add step to re-run cephadm playbook before deploy manila

### PR DESCRIPTION
**Pull request by seunghun1ee**

### CODE REVIEW

File: doc/source/contributor/environments/ci-multinode.rst [hunk 1]
```diff
@@ -30,8 +30,12 @@ is not enabled by default. To enable it, set the following in
       kolla_enable_manila: true
       kolla_enable_manila_backend_cephfs_native: true
 
-And re-run ``kayobe overcloud service deploy`` if you are working on an existing
-deployment.
+If you are working on an existing deployment, re-run Cephadm playbook by
+``kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/cephadm.yml``
+to add Ceph configurations (such as Manila pools and CephFS file system)
+for Manila.
```
Comment (Alex-Welsh) [inline doc/source/contributor/environments/ci-multinode.rst]: Just some minor wording suggestions:
```suggestion
If you are working on an existing deployment, re-run the Cephadm playbook
``kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/cephadm.yml``
to add the Ceph configuration (such as pools and CephFS file systems)
for Manila.
```

**Comment by MoteHue** [comment]
Does this need to be the whole playbook again? Takes quite a while, would be nice if we can just call the specific sub-playbook(s).

**Comment by seunghun1ee** [comment]
> Does this need to be the whole playbook again? Takes quite a while, would be nice if we can just call the specific sub-playbook(s).

Will be able to test this today. Plus, cephadm-gather-keys.yml also needs to be run. I'll update this PR.

**Review by Alex-Welsh — DISMISSED** [review]

File: doc/source/contributor/environments/ci-multinode.rst [hunk 2]
```diff
@@ -103,35 +110,36 @@ Then create a share type and share:
 
 .. code-block:: bash
 
-      manila type-create cephfs-type false --is_public true
-      manila type-key cephfs-type set vendor_name=Ceph storage_protocol=CEPHFS
-      manila create --name test-share --share-type cephfs-type CephFS 2
+      openstack share type create cephfs-type false --public true
+      openstack share type set cephfs-type --extra-specs vendor_name=Ceph
+      openstack share type set cephfs-type --extra-specs storage_protocol=CEPHFS
```
Comment (Alex-Welsh) [inline doc/source/contributor/environments/ci-multinode.rst]: nit, could've merged these into one line

Comment (seunghun1ee) [reply doc/source/contributor/environments/ci-multinode.rst]: I tested and it doesn't work with either ``openstack share type set cephfs-type --extra-specs vendor_name=Ceph,storage_protocol=CEPHFS`` or `` openstack share type set cephfs-type --extra-specs vendor_name=Ceph --extra-specs storage_protocol=CEPHFS``

Former one sets vendor_name "Ceph,storage_protocol=CEPHFS"
Latter one only sets storage_protocol

Do you know what correct syntax is?

Comment (MoteHue) [reply doc/source/contributor/environments/ci-multinode.rst]: [extra-specs](https://docs.openstack.org/python-manilaclient/latest/cli/osc/v2/index.html#share-type-set) is a list of key,value pairs: 
```
share type set
    [--extra-specs [<key=value> ...]]
    [--public <public>]
    [--description <description>]
    [--name <name>]
    <share_type>
```
So I reckon you'd be looking for: `openstack share type set cephfs-type --extra-specs vendor_name=Ceph, storage_protocol=CEPHFS`

Comment (MoteHue) [reply doc/source/contributor/environments/ci-multinode.rst]: Note the space after the comma :)

**Review by Alex-Welsh — APPROVED** [review]

**Pull request merged**